recipe/meta.yaml

```
@@ -21,22 +21,29 @@ package:
 
 source:
   - url: https://github.com/microsoft/onnxruntime/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 480ff5d2450ff18d02db48908c9c0c6a602993942cb47866b8effe32d0469a2c
+    sha256: 870ee3772d265d1c7b0799ac7280071f8ecd55b11ee3f11868a3e79c7f4cab0f
     patches:
-      - never_call_install_python_deps.patch
+      - dont-call-pip-on-win.patch
       # Workaround for https://github.com/conda-forge/onnxruntime-feedstock/pull/56#issuecomment-1586080419
       - windows_workaround_conflict_onnxruntime_dll_system32.patch  # [win]
       - cxx_17_for_darwin.patch
       # Workaround for https://github.com/conda-forge/onnxruntime-feedstock/pull/115#issuecomment-2122077816
       - do_not_pass_msvc_style_definitions_to_nvcc.patch  # [win]
-      - remove_numpy_version.patch
 
+      # https://github.com/conda-forge/onnxruntime-feedstock/pull/128#issuecomment-2390332504
+      - use_first_nvcc_found.patch
+
+      # Improve compatibility between Visual Studio 19.41 + cuda 12.0 (they require 12.4 by default)
+      # Upstream suggestion https://github.com/microsoft/onnxruntime/pull/22332
+      - 22332.patch
 
 build:
   number: {{ build }}
+  skip: true  # [not win]
+  skip: true  # [py!=311]
+  skip: true  # [win and suffix == "-novec"]
```

```suggestion
  # no windows user has requested novec (yet)
  skip: true  # [win and suffix == "-novec"]
```

its very likely that the builds will be successful.

If you are all OK with waiting for windows to get built "later" for CUDA 12.0 then feel commit this, and rerender (to create like 30 jobs) and merge the result, even if windows + cuda 12.0 will be red.